src/Nethermind/Nethermind.TxPool/ITxPoolConfig.cs

```diff
@@ -20,6 +20,11 @@ public interface ITxPoolConfig : IConfig
 
         [ConfigItem(DefaultValue = "null",
             Description = "Max transaction gas allowed.")]
+
         long? GasLimit { get; set; }
+
+        [ConfigItem(DefaultValue = "null",
+            Description = "Minutes between reporting on current state of tx pool.")]
+        int? ReportMinutes { get; set; }
```

Comment:
Do we need it configurable? We should have this in the actual metrics. It is fine for it to be in some Debug/Trace logs but other than that looks like overkill.

At minimum I would hide it from docs, as this has no use for users.

Reply:
People like them metrics 😅 I'll probably have it on all the time and currently logging in Nethermind isn't typed so you'd have to switch on a tsunami of output and then not be able to see it. So this is a way to switch it on independently.

It is useful as it tells you about the general state of the mem pool.

Reply:
> Looks good on costs with #5360 (merged into this branch) with 70% of txns being dropped in now light filters before it gets to heavier ones
> 
> <img alt="image" width="574" src="https://user-images.githubusercontent.com/1142958/222710806-422c493a-6fd3-40d2-bed1-4ba8c7f58cb9.png">

one more thing about TxPool filters - would caching balance an nonce somehow help there? Would be good to have some kind of histogram of incoming transactions senders between block (how many have unique senders vs how many senders are duplicated)